src/components/check-your-phone/index.njk

```diff
@@ -21,6 +21,8 @@
     <form action="/check-your-phone" method="post" novalidate="novalidate">
         <input type="hidden" name="_csrf" value="{{csrfToken}}"/>
         <input type="hidden" name="phoneNumber" value="{{phoneNumber}}"/>
+        <input type="hidden" name="resendCodeLink" value="{{resendCodeLink}}"/>
+        <input type="hidden" name="changePhoneNumberLink" value="{{changePhoneNumberLink}}"/>
```

Comment:
should the code be commented to explain WHY those fields are present and hidden?
It's obvious what the code is doing. but not why. 
code should be self explanatory what it's doing, comments are helpful to explain why it needs to do that thing